pkg/network/offsetguess.go

```diff
@@ -1,6 +1,6 @@
 // +build linux_bpf
```

Comment:
Thoughts on having this in eBPF as well?

Reply:
I don't have strong opinion about it. I don't think it feats neatly into `networks` but it feels very ad-hoc to be placed in `ebpf`. Up to you!